README.md

```diff
@@ -33,7 +33,7 @@ The following describes how to run the app in a Docker container by first buildi
 1. Run the container and run the app
 
     ```bash
-    docker run -it chappy bash
+    docker run -it -v /path/to/chappy:/app/chappy chappy bash
```

Comment:
Mounting is only needed for the development container. The instructions above are for when you want to just use the image you just created, which has the repo baked-in.

Reply:
This didnt work for me

Reply:
but yeah windows